index.html

```diff
@@ -0,0 +1,158 @@
+                <svg width="24" height="21" viewBox="0 0 24 21" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <rect width="26.4618" height="4.7339" transform="matrix(0.757375 0.65298 0.65298 -0.757375 0.18457 3.58533)" fill="#6070FF"/>
+                 </svg>     
+            </div>
+            <div class="bigec">
+                <svg width="52" height="101" viewBox="0 0 52 101" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M98.2411 26.9701C94.1949 19.7876 88.5141 13.659 81.6588 9.08033C74.8034 4.5017 66.9662 1.6018 58.7816 0.615434C50.5971 -0.370932 42.2953 0.583959 34.5484 3.40279C26.8016 6.22162 19.8274 10.8252 14.1906 16.8407C8.5538 22.8562 4.41283 30.1145 2.10297 38.0281C-0.206881 45.9416 -0.620695 54.2879 0.895036 62.3911C2.41077 70.4944 5.81344 78.1268 10.8275 84.6703C15.8416 91.2139 22.3262 96.4848 29.7563 100.056L39.5429 79.6952C35.31 77.6606 31.6158 74.6578 28.7592 70.9299C25.9027 67.2021 23.9642 62.8539 23.1007 58.2375C22.2372 53.6211 22.4729 48.8662 23.7889 44.3579C25.1048 39.8496 27.4639 35.7145 30.6752 32.2874C33.8864 28.8604 37.8596 26.2378 42.273 24.6319C46.6864 23.026 51.4159 22.482 56.0786 23.044C60.7414 23.6059 65.2062 25.258 69.1117 27.8664C73.0172 30.4748 76.2535 33.9663 78.5587 38.0582L98.2411 26.9701Z" fill="#6070FF"/>
+                    </svg>    
+            </div>
+            <div class="smallec">
+                <svg width="24" height="28" viewBox="0 0 24 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M-11.8095 0.641908C-13.8144 5.33769 -13.8718 10.6376 -11.969 15.3757C-10.0663 20.1137 -6.35924 23.9019 -1.66346 25.9068C3.03231 27.9117 8.3322 27.969 13.0703 26.0663C17.8084 24.1635 21.5965 20.4565 23.6014 15.7607L-11.8095 0.641908Z" fill="#6070FF"/>
+                </svg>                
+            </div>
+            <div class="emptycircle">
+                <svg width="15" height="15" viewBox="0 0 15 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <circle cx="7.5" cy="7.5" r="7" fill="white" stroke="#CBCDFF"/>
+                 </svg>              
+            </div>
+            <div class="circle">
+                <svg width="11" height="12" viewBox="0 0 11 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <ellipse cx="5.30398" cy="6.21088" rx="5.5" ry="5" transform="rotate(52.9708 5.30398 6.21088)" fill="#CBCDFF"/>
+                </svg>                
+            </div>
+            <div class="siderec">
+                <svg width="9" height="44" viewBox="0 0 9 44" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <rect x="-34" y="37.4868" width="51.9898" height="9.30073" transform="rotate(-46.1415 -34 37.4868)" fill="#EBEBFF"/>
+                 </svg>      
+            </div>
+        
+    </section>
+
+    <!-- the text part -->
+
+    <section class="text">
+        <h1>I’m Mariane
+            Glad to see you!
+        </h1>
```

Comment:
- [ ] As this is your portfolio I encourage you to use your name here instead of using the template's name, kindly mention your name here :smile: